Kconfig.zephyr

```diff
@@ -344,6 +344,25 @@ config BUILD_OUTPUT_S19
 	  Build an S19 binary zephyr/zephyr.s19 in the build directory.
 	  The name of this file can be customized with CONFIG_KERNEL_BIN_NAME.
 
+config BUILD_OUTPUT_UF2
+	bool "Build a binary in UF2 format"
+	default n
```

Comment:
Default is `n` so just remove.
```suggestion
```

Reply:
Removed. Thanks, I missed that earlier review comment.